components/kyma-metrics-collector/dev/deployment.yaml

```diff
@@ -30,7 +30,6 @@ spec:
           - --worker-pool-size=5
           - --log-level=debug
           - --listen-addr=8080
-          - --gardener-namespace=garden-kyma-dev
```

Comment:
Is this change needed?

Reply:
Yes, this flag was removed [here](https://github.com/kyma-project/control-plane/pull/3353/files#diff-ca7aca1159f0207308cbd2ac2674ab21e30376b4c39a2b52babe27ea766badd5L29)